docs/guide/extensibility/indentation.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Indentation
+---
+
+# Indentation
+
+Sublime Text can perform automatic indentation when you type.
+It can also reindent text according to the indentation rules on demand.
+Note that it will not perform auto-formatting, like adding newlines or adjusting spaces around parenthesis and the like. You would need an external tool or plugin for that.
```

Comment:
Please consider adding more line breaks at positions that make sense semantically. I assume you're still in the process of revising this, however, since some sections already use shorter lines.

Reply:
The plural is "parentheses."